dev/io.openliberty.checkpoint/bnd.bnd

```diff
@@ -17,7 +17,7 @@ Bundle-Description: Open Liberty Checkoint Support, version ${bVersion}
 Bundle-SymbolicName: io.openliberty.checkpoint
 Private-Package: io.openliberty.checkpoint.internal
 Export-Package: io.openliberty.checkpoint.internal.criu
-Import-Package: org.apache.felix.service.command; resolution:=optional, *
+Import-Package: org.apache.felix.service.command;version="[1.0,2)";resolution:=optional,*
```

Comment:
Is the version not filled in by bnd during the build?

Reply:
Unintentionally added to commit from a temp fix I made before you fixed the issue yourself. I will remove.